librz/analysis/cond.c

```diff
@@ -75,21 +75,21 @@ RZ_API int rz_analysis_cond_eval(RzAnalysis *analysis, RzAnalysisCond *cond) {
 	if (cond->arg[1]) {
 		st64 arg1 = (st64)rz_analysis_value_to_ut64(analysis, cond->arg[1]);
 		switch (cond->type) {
-		case RZ_ANALYSIS_COND_EQ: return arg0 == arg1;
-		case RZ_ANALYSIS_COND_NE: return arg0 != arg1;
-		case RZ_ANALYSIS_COND_GE: return arg0 >= arg1;
-		case RZ_ANALYSIS_COND_GT: return arg0 > arg1;
-		case RZ_ANALYSIS_COND_LE: return arg0 <= arg1;
-		case RZ_ANALYSIS_COND_LT: return arg0 < arg1;
+		case RZ_TYPE_COND_EQ: return arg0 == arg1;
```

Comment:
I recommend to extract this to `librz/type` as well, just make `arg0` and `arg1` function arguments.